atlasdb-ete-tests/src/main/java/com/palantir/atlasdb/todo/TodoClient.java

```diff
@@ -68,4 +91,59 @@ public void addTodo(Todo todo) {
                 .map(ImmutableTodo::of)
                 .collect(Collectors.toList());
     }
+
+    public void storeSnapshot(InputStream snapshot) {
```

Comment:
So each store marks the previous one as unused, so we always have at most one used stream?